.github/workflows/conda-forge.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       run: |
         cd build
         # ImuTest is skipped due to https://github.com/robotology/gz-sim-yarp-plugins/issues/54
-        ctest -E "^LaserTest|^CameraTest|^ImuTest|^ForceTorqueTest|^ControlBoardCommonsTest" --repeat until-pass:5 --output-on-failure -C ${{ matrix.build_type }} .
+        ctest -E "^LaserTest|^CameraTest|^ImuTest|^ForceTorqueTest|^ControlBoardCommonsTest|^ConfigurationParsingTest" --repeat until-pass:5 --output-on-failure -C ${{ matrix.build_type }} .
```

Comment:
If we skip a test, it is a good idea to add a comment explaining why we are skipping it (such as the one already present before this command).

Reply:
Done in e452f8a1261abcd2ef02039660be3293f9098c78